### ISSUE TITLE
neuprintr:::neuprint_name_field fails for VNC

### ISSUE THREAD

**Issue by jefferis** [natverse/neuprintr#139]
@dokato this is the source of your other bug report.

**Comment by jefferis** [comment]
https://github.com/natverse/neuprintr/blob/64bebfbaa93e23745f080ba6aeb82c3ec4b4c5a7/R/fetch.R#L111-L122

**Comment by jefferis** [comment]
@dokato 

```r
q = "MATCH (n :hemibrain_Neuron) WHERE exists(n.instance) RETURN count(n)"
```

should be 

```r
q = "MATCH (n :Neuron) WHERE exists(n.instance) RETURN count(n)"
```
could you test if that fixes and make a PR?

**Comment by jefferis** [comment]
Closed by #140
